v2/googlecloud-to-elasticsearch/docs/GCSToElasticsearch/README.md

```diff
@@ -72,7 +97,7 @@ echo '{
     "image":"'${TARGET_GCR_IMAGE}'",
     "metadata":{
       "name":"GCS to Elasticsearch",
-      "description":"Replicates from a file stored in GCS into Elasticsearch",
+      "description":"Ingests data from files stored in GCS into Elasticsearch",
       "parameters":[
```

Comment:
isOptional should be true for below parameters
elasticsearchUsername
elasticsearchPassword
delimiter

apiKey should be added to parameters